_pages/en/it-picture-medium-term.md

```diff
@@ -26,124 +26,131 @@ The foundational pieces for "Government as a service" vision are available and i
+ESDC is a leader in OneGC Council and Clusters.
+Its own digital transformation initiative is going well with Benefits Delivery Modernization (BDM) and the Service Transformation progressing according to plan.
 
 ### IT within ESDC
 
 IT is used as a strategic asset to improve services to citizens and is no longer seen as a "back-end" function.
-ESDC service lines understand the cost and risks of using IT and have started to share its accountability by using a different governance and investment method as a means to reduce IT risks.
-This means part of their budget is set to assure technical debt is exposed and handled in a timely manner.
-IT fully understands its impact on ESDC's mandate and has heavily invested in learning, agility and automation as means to respond faster to business change.
+ESDC service lines understand the cost and risks of leveraging IT and have started to share its accountability by using a different governance and investment method as a means to reduce IT-related risks.
+This means part of their budget is set to ensure technical debt is documented and handled in a timely manner.
+IT fully understands its impact on ESDC's mandate and has heavily invested in Learning, Agility and Automation as means to respond faster to business changes.
 
-The ESDC IT Strategy put in place a continuous improvement [transformation](https://cloud.google.com/solutions/devops/devops-culture-transform) based on the capabilities and recommendations from the DevOps Research and Assessment (DORA) and the annual [State of DevOps Report](https://cloud.google.com/devops/).
-The goal has been to continuously increase frequency of delivery while reducing the lead time, the time to restore service when an incident occurs and the rate of failure.
+The ESDC IT Strategy have put in place Continuous Improvement [Transformation](https://cloud.google.com/solutions/devops/devops-culture-transform) based on the capabilities and recommendations of the DevOps Research and Assessment (DORA) and the annual [State of DevOps Report](https://cloud.google.com/devops/).
+The goal has been to continuously increase frequency of delivery while reducing lead time, time to restore service when an incident occurs and rate of failure.
 
 The continuous improvement transformation is for all things that IT is responsible for.
-This includes applications (software delivery, operations), infrastructure (user devices, services, networks).
+This includes applications (software delivery, operations) and infrastructure (user devices, services, networks).
 
 #### Cultural
 
-- https://cloud.google.com/solutions/devops/devops-culture-learning-culture
-- https://cloud.google.com/solutions/devops/devops-culture-job-satisfaction
-- https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture
+- [https://cloud.google.com/solutions/devops/devops-culture-learning-culture](https://cloud.google.com/solutions/devops/devops-culture-learning-culture)
```

Comment:
I think these need to be linked into sections of the paragraph? Did I get that properly, @smellems?